ROM.cs

```diff
@@ -50,16 +49,18 @@ private static int AddNewFile(string filename)
             return newfile.Addr;
         }
 
-        private static int AddrToFile(uint RAddr)
+        private static int AddrToFile(int RAddr)
         {
             int i = 0;
-            for (; i < MMFileList.Count; i++)
+            while (i < MMFileList.Count)
```

Comment:
Could this function be basically `return MMFileList.FindIndex(file => RAddr => file.Addr && RAddr < file.End)`

Reply:
Though actually the results would be different if the RAddr is not found. Currently it would return `MMFileList.Count` whereas what i said would return `-1`. Would that be a problem?